lib/internal/process/pre_execution.js

```diff
@@ -119,6 +124,14 @@ function prepareExecution(options) {
   }
 }
 
+function setupSymbolDisposePolyfill() {
+  // TODO(MoLow): Remove this polyfill once Symbol.dispose and Symbol.asyncDispose are available in V8.
+  // eslint-disable-next-line node-core/prefer-primordials
+  Symbol.dispose ??= SymbolDispose;
+  // eslint-disable-next-line node-core/prefer-primordials
+  Symbol.asyncDispose ??= SymbolAsyncDispose;
```

Comment:
Would this be a good thing to polyfill them? Wouldn't users assume certain behaviors from the runtime because they exists?

Reply:
I don't think there is a way for typescript or bable to recognize these symbols without them being global.

Reply:
@mcollina this is confirmed (with the TS team) to play nice. Users would be able to use them with the polyfill but not the syntactic sugar until v8 ships

Reply:
That breaks `core-js`.

Reply:
https://github.com/nodejs/node/issues/48699